src/testing/factories/juju/Application.ts

```diff
@@ -1,33 +1,34 @@
 import type {
-  AdditionalProperties,
   ApplicationGetResults,
   Base,
   Value,
   ErrorResults,
 } from "@canonical/jujulib/dist/api/facades/application/ApplicationV15";
```

Comment:
I've looked at the source code of `ApplicationV18` and it should work instead of `ApplicationV15`, while being the up to date version (as far as I understand). The only difference I've observed between V15 and V18 is that the `Base` type has an extra key `"image-id"`, which I am not sure how much would affect the dashboard.

Reply:
Well spotted, as you say this won't have much impact so I've updated these to the latest types.